src/lib/formats/ap2_dsk.h

```diff
@@ -21,12 +21,9 @@
 
 ***************************************************************************/
 
-#define APPLE2_NIBBLE_SIZE 416
-#define APPLE2_SMALL_NIBBLE_SIZE 374
-#define APPLE2_STD_TRACK_COUNT 35
-#define APPLE2_TRACK_COUNT 40
-#define APPLE2_SECTOR_COUNT 16
-#define APPLE2_SECTOR_SIZE 256
+constexpr int APPLE2_STD_TRACK_COUNT = 35;
+constexpr int APPLE2_TRACK_COUNT = 40;
+constexpr int APPLE2_SECTOR_SIZE = 256;
```

Comment:
Can the scope of these be reduced?  Do they need to be in the global namespace?  Are the nibble sizes useful for documentation even if they’re currently unused?

Reply:
> Can the scope of these be reduced? Do they need to be in the global namespace?

I thought about moving them to the `.cpp` file, but since I used them in the definition of `a2_nib_format`, I can't do that anymore. (Technically, I could still move `APPLE2_SECTOR_SIZE`, but it seems better for readability to keep these constants together.)

> Are the nibble sizes useful for documentation even if they’re currently unused?

I don't even know what these values are supposed to be, to be honest. I thought they were the sector data sizes after GCR encoding (in the 13-sector and 16-sector formats), but on second look, that doesn't seem to actually be the case.